# HG changeset patch
# User Rik <rik@octave.org>
# Date 1504223672 25200
#      Thu Aug 31 16:54:32 2017 -0700
# Node ID 8db08d3f48223dae669c579d0dd4d3a070273573
# Parent  aa127fce67e4437e3aff995458dc69fca8abf803
Don't mark bug-38236 as a regression (bug #38236).

* test/bug-38236/bug-38236.tst: Remove '*' from BIST test number.

diff --git a/test/bug-38236/bug-38236.tst b/test/bug-38236/bug-38236.tst
--- a/test/bug-38236/bug-38236.tst
+++ b/test/bug-38236/bug-38236.tst
@@ -11,10 +11,10 @@
 ## WITHOUT ANY WARRANTY; without even the implied warranty of
 ## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 ## GNU General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
-%!test <*38236>
+%!test <38236>
 %! u_vr
